### PR TITLE
[1.67.0] 32090: Add info about appVersionLabel to enterprise install docs

### PR DESCRIPTION
**Pull request by jonquil2002**
SC: https://app.shortcut.com/replicated/story/32090/ability-to-install-a-specific-prior-app-version-for-embedded-and-existing-cluster-online-installs

Previews:

- https://deploy-preview-236--replicated-docs.netlify.app/enterprise/installing-existing-cluster-online#install-with-the-app-manager

- https://deploy-preview-236--replicated-docs.netlify.app/enterprise/installing-embedded-cluster

### CODE REVIEW

File: docs/enterprise/installing-embedded-cluster.md [hunk 1]
```diff
@@ -10,12 +10,16 @@ Before you install, ensure that you meet the system requirements. For more infor
...
 
-To install the admin console on a cluster created by the Kubernetes installer, run the installation script provided by the application developer.
+To install the admin console on a cluster created by the Kubernetes installer, run the installation script provided by the application vendor.
 
 ```bash
 curl -sSL https://kurl.sh/supergoodtool | sudo bash
 ```
 
+:::note
+With KOTS v1.67.0 and higher, you can install a specific version of the application. Your application vendor provides an installation script with the `appVersionLabel` flag set to the specific version. Example: `curl https://kurl.sh/supergoodtool | sudo bash -s appVersionLabel=3.1.0`.
```
Comment (ajp-io) [inline docs/enterprise/installing-embedded-cluster.md:20]: WDYT about this? Similar comment could be made for the existing cluster note too.
```suggestion
With KOTS v1.67.0 and later, you can install a specific version of the application. Use the `appVersionLabel` flag and the version label for a particular version of your vendor's application. Example: `curl https://kurl.sh/supergoodtool | sudo bash -s appVersionLabel=3.1.0`.
```

**Review by ajp-io — APPROVED** [review]

**Comment by jonquil2002** [comment]
Great suggestions, @ajp-io .

@paigecalvert Ready for your review/approval :)

**Review by paigecalvert — APPROVED** [review]
Looks great!

**Comment by ajp-io** [comment]
We'll wait to merge this until the corresponding functionality is released in KOTS.

**Pull request merged**